.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: New Command request
+about: Ask for a new command
+title: 'New Command'
+labels: 'enhancement'
+
+---
+
+| Questions         | Answers
+| ----------------- | -------------------------------------------------------
+| Command name?      | the command name "fop:domain:mycommand"
+| Command domain?  | the command domain ( configuration / images / cache / dev ... ) 
+| Command arguments?   | List the command arguments
+| Command options?   | List the command options 
+| Command return? | List what the command will return <br > success : _what command retrun in success_ <br /> error : _what command return in error_ <br> void if nothing
```

Comment:
do you mean command effects or command return message ?

You can also modify, add a missing 's' : what command returnS

Reply:
I agree it is a lot to ask :smile: 
To my point of view the more information we have the better it is, but you're right that for normal user it may be too much.
I can remove it.

Reply:
I suggest to remove 'command return' and to merge 'arguments' and 'options' under 'parameters'.
This saves 2 lines.

Reply:
The change is done :smiley: